src/components/Gradebook/index.jsx

```diff
@@ -274,6 +284,26 @@ export default class Gradebook extends React.Component {
     }
   };
 
+  handleSubmitAssignmentGrade = (event) => {
+    event.preventDefault();
+    const formContents = new FormData(event.target);
```

Comment:
I'm pretty confident we still support it, certainly for learners. Thanks for the reminder! (FormData isn't altogether verboten for IE11-supporting browsers, but you're right that the functionality I'm using here is! In other situations it can still be used in a cross-browser way to customize form submission behavior without losing the benefit of how the browser will package up the form data for you.) Thanks for pointing this out; strong motivation for making that change to a state data source